tests/keras/utils/vis_utils_test.py

```diff
@@ -11,7 +11,6 @@
 from keras.utils import vis_utils
 
 
-@pytest.mark.skipif(sys.version_info > (3, 0), reason='pydot-ng currently supports python 3.4')
```

Comment:
The tests are failing. Would you please add `pip install pydot graphviz;` to the line 71 on `.travis.yml`?